ci/build/npm-postinstall.sh

```diff
@@ -23,8 +23,8 @@ main() {
   # Grabs the major version of node from $npm_config_user_agent which looks like
   # yarn/1.21.1 npm/? node/v14.2.0 darwin x64
   major_node_version=$(echo "$npm_config_user_agent" | sed -n 's/.*node\/v\([^.]*\).*/\1/p')
-  if [ "$major_node_version" -lt 12 ]; then
-    echo "code-server currently requires at least node v12"
+  if [ "$major_node_version" -lt 14 ]; then
```

Comment:
We talked about this elsewhere, for those reading I think if we don't restrict this we will eventually have people submitting Node version-related issues.

But then again it does feel to me like it goes against the spirit of installing with yarn/npm.

So what if we add the version of node to the output of `code-server --version` that way when people submit issues using unsupported versions of Node we can easily say "please try a supported version" or something like that?

Reply:
Actually we can add it to the output of `code-server --version --json` and switch the issue template to use that. I don't want to change the output of `--version` since people might be using that output for things.

Reply:
One more comment, I'm down to leave that for another PR but it's pretty straightforward so if we want to do that here that sounds fine to me too.

Reply:
We don't really provide any CLI stability guarantees, AFAIU, and people parsing stuff *should* be using `--json` anyway...

Reply:
I think another PR would be best - don't want to collect too many changes here

Reply:
I agree but let's not break it anyway :smile: 

All right, we should probably remove this check entirely so the user can use whatever version they want right?

Reply:
I'm generally okay with a minimum version, since updating is always an option (afaik); but I'll leave that to you

Reply:
Hmm I'm not sure I feel too strongly about it. I suppose my thinking is that if we don't want to cap the version because users might have a different version of Node installed then that applies to older versions as well.

Reply:
Another update, we're going to restrict to 14 since we know that works but have an env var that allows users to run with a different version so this should check all the boxes.